src/java.desktop/share/classes/java/awt/doc-files/AWTThreadIssues.html

```diff
@@ -154,11 +154,11 @@ <h3>
 <ul>
   <li> Other packages can create displayable components for internal
        needs and never make them undisplayable. See
-<a href="https://bugs.java.com/view_bug.do?bug_id=4515058">
+<a href="https://bugs.openjdk.org/browse/JDK-4515058">
 4515058</a>,
-<a href="https://bugs.java.com/view_bug.do?bug_id=4671025">
+<a href="https://bugs.openjdk.org/browse/JDK-4671025">
 4671025</a>, and
-<a href="https://bugs.java.com/view_bug.do?bug_id=4465537">
+<a href="https://bugs.openjdk.org/browse/JDK-4465537">
```

Comment:
This bug was about some off-screen image problem with JScrollBar
It was closed as a dup of 4417287 which was fixed in 1.4

So in summary it is wrong to keep these linked bugs in here.
Remove the lot of them